AWS/legos/aws_list_all_iam_users/aws_list_all_iam_users.py

```diff
@@ -0,0 +1,33 @@
+##
+# Copyright (c) 2021 unSkript, Inc
+# All rights reserved.
+##
+from pydantic import BaseModel, Field, SecretStr
+from typing import Dict, List
+import pprint
+
+class InputSchema(BaseModel):
+    pass
+
+def aws_list_all_iam_users_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+def aws_list_all_iam_users(handle) -> List:
+    """aws_list_all_iam_users lists all the IAM users
+
+        :type handle: object
+        :param handle: Object returned from Task Validate
+        
+        :rtype: Result List of result
```

Comment:
You meant Result List of Regions?

Reply:
or users?

Reply:
Of users. Made the changes, please check